packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -61,6 +62,30 @@ const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
 	);
 };
 
+interface TrackDesignViewProps {
+	categorization?: string | null;
+	design: Design;
+	isPremiumThemeAvailable?: boolean;
+}
+
+const TrackDesignView: React.FC< TrackDesignViewProps > = ( {
+	categorization,
+	design,
+	isPremiumThemeAvailable,
+} ) => {
+	useEffect( () => {
```

Comment:
I'd prefer to wrap it into a hook named `useTrackDesignView` instead of the component